unidist/cli/backends/ray/runner.py

```diff
@@ -0,0 +1,65 @@
+# Copyright (C) 2021 Modin authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+"""``RayRunner`` class functionality."""
+
+import os
+import warnings
+
+from unidist.cli.base.runner import BackendRunner
+from unidist.cli.base.utils import Defaults, BackendName, validate_num_cpus
+
+
+class RayRunner(BackendRunner):
+    """
+    An implementation of unidist ``BackendRunner`` for Ray backend.
+
+    Parameters
+    ----------
+    **cli_kwargs : dict
+        Keyword arguments supported by unidist CLI.
+    """
+
+    def __init__(self, **cli_kwargs):
+        self.backend = BackendName.RAY
+        super().__init__(**cli_kwargs)
+
+    def check_kwargs_support(self, **kwargs):
+        """Check support for `kwargs` combination for Ray backend."""
+        hosts = kwargs.get("hosts", Defaults.HOSTS)
+        num_cpus = kwargs.get("num_cpus", Defaults.NUM_CPUS)
+        self.redis_password = kwargs.get("redis_password", Defaults.REDIS_PASSWORD)
```

Comment:
self.hosts, self.num_cpus and self.redis_password should be initialized by default values in the constructor.

Reply:
Initialization of these values with defaults is placed in base `BackedRunner` now.